Kernel/Devices/Storage/USB/SCSIComands.h

```diff
@@ -13,24 +13,145 @@
+    };
+    struct {
+        DeviceType device_type : 5;
+        u8 qualifier : 3;
+    } peripheral_info;
+    u8 removable; // 0x80 for removable, 0x00 for fixed
+    u8 version;
+    struct {
+        u8 response_data_format : 4;
+        u8 hierarchical_support : 1; // HISUP
+        u8 normal_aca_support : 1;   // NORMACA
+        u8 obsolete : 2;
+    } response_data;
+    u8 additional_length; // N-4
+    struct {
+        u8 protect : 1;
+        u8 reserved : 2;
+        u8 third_party_copy : 1;
+        u8 target_port_group_support : 2;
+        u8 access_control_coordinator : 1;
+        u8 scc_support : 1;
+
+        u8 : 4;
+        u8 multi_port : 1;
+        u8 vendor_specific1 : 1;
+        u8 enclosure_services : 1;
+        u8 : 1;
+
+        u8 vendor_specific2 : 1;
+        u8 command_queueing : 1;
+        u8 : 6;
+    } capabilities;
+
+    char vendor_id[8];
+    char product_id[16];
+    char product_revision_level[4];
+
+    StringView device_type_string() const
+    {
+        switch (peripheral_info.device_type) {
+        case DeviceType::DirectAccessBlockDevice:
+            return "Direct Access Block Device"sv;
+        case DeviceType::SequentialAccessDevice:
+            return "Sequential Access Device"sv;
+        case DeviceType::PrinterDevice:
+            return "Printer Device"sv;
+        case DeviceType::ProcessorDevice:
+            return "Processor Device"sv;
+        case DeviceType::WriteOnceDevice:
+            return "Write Once Device"sv;
+        case DeviceType::CDDVDDevice:
+            return "CD/DVD Device"sv;
+        case DeviceType::OpticalMemoryDevice:
+            return "Optical Memory Device"sv;
+        case DeviceType::MediumChangerDevice:
+            return "Medium Changer Device"sv;
+        case DeviceType::StorageArrayControllerDevice:
+            return "Storage Array Controller Device"sv;
+        case DeviceType::EnclosureServicesDevice:
+            return "Enclosure Services Device"sv;
+        case DeviceType::SimplifiedDirectAccessDevice:
+            return "Simplified Direct Access Device"sv;
+        case DeviceType::OpticalCardReaderWriterDevice:
+            return "Optical Card Reader/Writer Device"sv;
+        case DeviceType::BridgeControllerCommands:
+            return "Bridge Controller Commands"sv;
+        case DeviceType::ObjectBasedStorageDevice:
+            return "Object Based Storage Device"sv;
+        case DeviceType::AutomationDriveInterface:
+            return "Automation Drive Interface"sv;
+        case DeviceType::WellKnownLogicalUnit:
+            return "Well Known Logical Unit"sv;
+        case DeviceType::UnknownOrNoDeviceType:
+            return "Canonical Unknown or No Device Type"sv;
+        default:
+            return "Unknown Device Type"sv;
+        }
+    }
 };
-static_assert(AssertSize<ReadCapacity10Parameters, 8>());
+static_assert(AssertSize<StandardInquiryData, 36>());
 
-// 3.16
+// 3.16.1
```

Comment:
I can't find 3.16.1 in the Seagate manual?

Reply:
Weird that this command does not use subsections